packages/server/test/integration/cypress_spec.js

```diff
@@ -449,10 +452,13 @@ describe('lib/cypress', () => {
       })
     })
 
-    it('scaffolds out support + files if they do not exist', function () {
+    // NOTE: This test is skipped because it causes the subsequent test to fail intermittently
+    it.skip('scaffolds out support + files if they do not exist', async function () {
```

Comment:
@lmiller1990 @estrada9166 I marked this as skipped because it was causing the test after it to intermittently timeout in its current state. Maybe we could meet to discuss what this test actually does? In its current state I don't think we even make it to the block that calls `cypress.start`, so I'm not sure what this is actually testing.

Reading through the code here doesn't make much sense to me. It looks like we're actually asserting that support + files **are not** scaffolded despite what the test title says.

Reply:
I think we can nuke this, we don't auto scaffold support etc anymore, that is part of the launchpad onboarding. The fact this passes at all is probably because we stub a bunch of things? Weird.